.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,14 @@
+Fixes # .
+
+Describe your changes.
+
+# Review Checklist
+
+- [ ] Does it work in all major browsers: Firefox, Chrome, Edge, Safari?
+- [ ] Does it work on mobile?
+- [ ] Can you `rm -rf node_modules/ && npm install` and app works as expected?
+- [ ] Did you deploy a staging branch?
+
+## Optional
+
+- [ ] If it's a new component, is it in ES6? Is it clear of warnings from ESLint?
```

Comment:
Add this?
`[ ] Have you replaced any ChangeListener or PromiseRenderer components with code that updates component state?`